README.md

```diff
@@ -265,58 +264,7 @@ Finally there is [`heroic-dist`](heroic-dist), a small project that depends on a
 or through an interactive shell [`com.spotify.heroic.HeroicShell`](heroic-shell/src/main/java/com/spotify/heroic/HeroicShell.java).
 The shell can either be run standalone or connected to an existing Heroic instance for administration.
 
-#### Bypassing Validation
-
-To bypass automatic formatting and checkstyle validation you can use the
-following stanza:
-
-```java
-// @formatter:off
-final List<String> list = ImmutableList.of(
-   "Welcome to...",
-   "... The Wild West"
-);
-// @formatter:on
-```
-
-To bypass a FindBugs error, you should use the `@SupressFBWarnings` annotation.
-
-```java
-@SupressFBWarnings(value="FINDBUGS_ERROR_CODE", justification="I Know Better Than FindBugs")
-public class IKnowBetterThanFindbugs() {
-    // ...
-}
-```
-
-## HeroicShell
```

Comment:
described more at https://spotify.github.io/heroic/docs/shell so deleted it here